stack.yaml

```diff
@@ -43,33 +46,57 @@ extra-deps:
   - micro-recursion-schemes-5.0.2.2
   - moo-1.2
   - network-3.1.1.1
+  - persistent-2.10.5.1
+  - persistent-postgresql-2.10.1.2
+  - persistent-template-2.8.2.3
+  - pipes-safe-2.3.2
+  - primitive-0.7.1.0
+  - process-1.6.8.2
   - quiet-0.2
+  - quickcheck-instances-0.3.19
   - snap-core-1.0.4.1
   - snap-server-1.1.1.1
+  - sop-core-0.5.0.1
   - statistics-linreg-0.3
   - streaming-binary-0.3.0.1
-  - systemd-2.3.0
   - tasty-hedgehog-1.0.0.2
   - text-1.2.4.0
-  - text-ansi-0.1.0
   - text-zipper-0.10.1
+  - th-abstraction-0.3.2.0
+  - th-lift-0.8.1
   - th-lift-instances-0.1.14
+  - these-1.1
+  - time-1.8.0.4
+  - time-compat-1.9.3
   - time-units-1.0.0
   - transformers-except-0.1.1
-  - Unique-0.4.7.6
-  - word-wrap-0.4.1
+  - typerep-map-0.3.3.0
+  - unix-2.7.2.2
+  - webdriver-0.9.0.1
   - websockets-0.12.6.1
+  - Win32-2.8.5.0
+  - word-wrap-0.4.1
+
+  - git: https://github.com/erebe/systemd
+    commit: dba4b563dac99ff52328db916d99d928bc4a80fb
+
+  - git: https://github.com/CodiePP/cborg.git
+    commit: f6a2b7cf5ff4f515accaf026c95b4992d9f771bf
+    subdirs:
+        - cborg
+        - serialise
 
   - git: https://github.com/input-output-hk/cardano-ledger-specs
-    commit: 74188f62ff743c690bdf287f7fa18eaa2ee354d4
+    commit: 08f22528915bd2e125f20387dd171ba0f4e09742
```

Comment:
This commit is not on master, just in an unmerged PR. If you close that PR, rebase it, or it is rebased/squashed when merged, that commit will eventually disappear, breaking the build.

Reply:
right. this is not necessary anymore.